src/index.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
```

Comment:
I'd have a look further down this file about the colour scheme. I can't see any text on your site because I use dark mode.